assets/dev/js/frontend/handlers/playing-cards.js

```diff
@@ -0,0 +1,31 @@
+export default class PlayingCards extends elementorModules.frontend.handlers.Base {
+	getDefaultSettings() {
+		return {
+			selectors: {
+				playingCard: '.elementor-playing-card',
+			},
+		};
+	}
+
+	getDefaultElements() {
+		const selectors = this.getSettings( 'selectors' );
+		return {
+			$playingCard: this.$element.find( selectors.playingCard ),
+		};
+	}
+
+	bindEvents() {
+		this.elements.$playingCard.on( 'click', this.onCardClick.bind( this ) );
+	}
+	unbindEvents() {
+		this.elements.$playingCard.on( 'click', this.onCardClick.unbind( this ) );
```

Comment:
You are not unbinding the same event you are binding. 
Suggestion:

```
onInit(){
		super.onInit();
		this.onCardClick = this.onCardClick.bind( this )
}

bindEvents() {
		this.elements.$playingCard.on( 'click', this.onCardClick );
}
	
bindEvents() {
		this.elements.$playingCard.off( 'click', thisonCardClick);
}
```

Reply:
lol you absolutely right